inst/table.m

```diff
@@ -424,19 +424,26 @@ function prettyprint (this)
     ## -*- texinfo -*-
     ## @node table.size
     ## @deftypefn {Method} {@var{sz} =} size (@var{obj})
+    ## @deftypefnx {Method} {[@var{nr}, @var{nv}] =} size (@var{obj})
     ##
     ## Gets the size of a table.
     ##
     ## For tables, the size is [number-of-rows x number-of-variables].
     ## This is the same as @code{[height(obj), width(obj)]}.
     ##
     ## @end deftypefn
-    function out = size (this, dim)
-      if nargin == 1
-        out = [height(this), width(this)];
+    function [out, nv] = size (this, dim)
```

Comment:
I think this needs to become `varargout = ...` instead of a fixed list of argouts.